### ISSUE TITLE
🔒 Security: Relies on minimatch@3.0.4 version with minimatch ReDoS vulnerability

### ISSUE THREAD

**Issue by creharmony** [mochajs/mocha#4937]
Hi

I'm not expert of JS deps, but it seems that 10.1.0 rely on vulnerable minimatch dep version 3.0.4
```
└─┬ mocha@10.1.0
  ├─┬ glob@7.2.0
  │ └── minimatch@3.0.4

minimatch  <3.0.5
Severity: high
minimatch ReDoS vulnerability - https://github.com/advisories/GHSA-f8q6-p94x-37v3
```

cf. creharmony/node-etsy-client#56 and fix creharmony/node-etsy-client#57

It would be great to increase minimatch version [from mocha](https://github.com/mochajs/mocha/blob/master/package.json#L64) ? via `"glob": "7.2.0"` ?

NB: the [last glob version](https://github.com/isaacs/node-glob/blob/main/package.json#L23) rely on more recent version of minimatch

**Comment by joewagner** [comment]
I just found this same vulnerability in minimatch <=3.0.4.  I believe this lib should upgrade glob to >=8.0.0 if possible.

**Comment by WikiRik** [comment]
Duplicate of https://github.com/mochajs/mocha/issues/5009 (or rather that one is a duplicate of this earlier issue)
Seems to be mostly fixed in the master branch already, where glob is updated to 8.1.0

**Comment by JoshuaKGoldberg** [comment]
Released in [`mocha@v10.6.0`](https://github.com/mochajs/mocha/releases/tag/v10.6.0). A fresh install of `npm i mocha -D` on a clean directory only has `minimatch@5` now. 🚀
